tests/app/dao/test_service_sms_sender_dao.py

```diff
@@ -0,0 +1,16 @@
+from app.dao.service_sms_sender_dao import update_service_sms_sender
+from app.models import ServiceSmsSender
+from tests.app.db import create_service
+
+
```

Comment:
Should we have a test that checks we create a row if one doesn't already exist?

Reply:
That check is in the create service. Maybe I  should move this method.

Reply:
Oh, so are we testing it elsewhere?